.github/CHANGELOG.md

```diff
@@ -1,18 +1,10 @@
-# Release 0.19.0-dev
-
-### New features since last release
-
-### Breaking changes
+# Release "0.19.0"
```

Comment:
```suggestion
# Release 0.19.0
```

Reply:
Seems to be related to this line, correct?
https://github.com/PennyLaneAI/pennylane-sf/blob/master/.github/workflows/vb_script.py#L16

Reply:
Ahh great catch! Not quite! It is a little more complicated since the `_version.py` file requires the quotes. I will make a patch for it once I finish with the demos